Dockerfile-server

```diff
@@ -26,7 +26,7 @@ ARG IMAGE_SOURCE="Not specified"
 COPY cmd/ ./cmd
```

Comment:
Need to remove the `ARG IMAGE_CREATED="Not specified"` that is 2 lines up as it is no longer needed. Will lower our layers by 1